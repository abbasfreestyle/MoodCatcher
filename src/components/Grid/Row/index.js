import React from 'react';
import { View } from 'react-native';

const Row = props => <View flexDirection="row" {...props} />;

export default Row;
