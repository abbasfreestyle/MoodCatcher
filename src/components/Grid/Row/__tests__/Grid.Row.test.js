import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Row from '..';

const wrapper = shallow(<Row />);

describe('Grid > Row.', () => {
  describe('Snapshots.', () => {
    test('should match', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
