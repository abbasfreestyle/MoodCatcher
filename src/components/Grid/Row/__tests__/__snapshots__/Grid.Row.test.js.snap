// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid > Row. Snapshots. should match 1`] = `
<View
  flexDirection="row"
/>
`;
